### PR TITLE
TF-1545: Update pick color dialog in rich text composer

### PR DESCRIPTION
**Pull request by nqhhdev**
Issue: #1545 

`Resolved` 

I've contributed with a PR to update this, keep an eye on it and wait for their response

- https://github.com/rydmike/flex_color_picker/pull/65
 

https://user-images.githubusercontent.com/99852347/229022199-74081c4b-ee52-4394-b021-bc1580f6640a.mov

### CODE REVIEW

**Comment by chibenwa** [comment]
IMO we should not allow the color picker to go too wide...

## Good

On small size screen colors are logically alligned making it easier to pick one:

Screenshot from 2023-03-31 08-32-58

## Bad

On wider screens it just becomes a mess...

Screenshot from 2023-03-31 08-33-12

## Suggestion

Do not allow the color picker to grow too wide...

**Comment by dab246** [comment]
- Should center the palette.
- Padding=16 has all (top, botto, lef, right)

**Comment by nqhhdev** [comment]
> * Should center the palette.
> * Padding=16 has all (top, botto, lef, right)
> 
> Screenshot 2023-03-31 at 11 39 42

Screenshot 2023-03-31 at 13 13 02
I updated it

File: pubspec.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ dependencies:
   html_editor_enhanced:
     git:
       url: https://github.com/linagora/html-editor-enhanced.git
-      ref: email_supported
+      ref: change-flex-color-picker-lib
```
Comment (dab246) [inline pubspec.yaml]: why updated this library?

**Review by dab246 — APPROVED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (chibenwa) [reply pubspec.yaml]: Should be `master`

Comment (nqhhdev) [reply pubspec.yaml]: -`html-editor-enhanced=`This library also uses `html-editor-enhanced`. So I switched to my branch for testing before being merged into master. I updated it again

Plss check my PR: https://github.com/linagora/html-editor-enhanced/pull/9

**Pull request merged**